Assets/SEE/UI/PropertyDialog/UserPropertyDialog.cs

```diff
@@ -0,0 +1,162 @@
+﻿using SEE.Controls;
+using SEE.UI.Notification;
+using System;
+using SEE.Utils;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace SEE.UI.PropertyDialog
+{
+    internal class UserPropertyDialog
+    {
+        /// <summary>
+        /// Callback to be called when this dialog closes.
+        /// </summary>
+        public delegate void OnClosed();
```

Comment:
Instead of defining this delegate, consider using `Action` instead.

Reply:
I used this way because it is the same as in NetworkPropertyDialog. Should i change both classes?

Reply:
Yes, please do.

Reply:
fixed this

Reply:
Looks good to me!